package.json

```diff
@@ -11,7 +11,7 @@
     "@asyncapi/converter": "^1.1.0",
     "@asyncapi/diff": "^0.4.1",
     "@asyncapi/generator": "^1.9.12",
-    "@asyncapi/modelina": "^1.0.0-next.40",
+    "@asyncapi/modelina": "^1.0.0-next.43",
```

Comment:
please use latest modelina, main release not release candidate 

it was released few days ago

Reply:
https://github.com/asyncapi/modelina/releases/tag/v1.1.1

Reply:
ok, I see you changed it but there are anyway conflicts you need to solve as there were some releases in CLI yesterday

Reply:
Ok I will look into this one